test/parallel/test-webcrypto-derivebits-cfrg.js

```diff
@@ -52,7 +52,7 @@ async function prepareKeys() {
           Buffer.from(spki, 'hex'),
           { name },
           true,
-          ['deriveKey', 'deriveBits']),
+          []),
```

Comment:
Is this change semver-major? Now that Web Crypto is stable, we shouldn't land breaking change on it without treating it as semver-major.

Reply:
CFRG curves remain experimental